chartfx-chart/src/test/java/de/gsi/chart/renderer/spi/financial/utils/CalendarUtils.java

```diff
@@ -114,4 +114,27 @@ public static Calendar createByDateTime(String datetimePattern) throws ParseExce
 
         return cal;
     }
+
+    /**
+     * Create the calendar instance by datetime pattern:
+     * MM/dd/yyyy HH:mm
+     * for example: 12/01/2017 15:30
+     *
+     * @param datetimePattern String
+     * @return calendar interval instance
+     * @throws ParseException parsing fails
+     */
+    public static Calendar createByTradeStationDateTime(String datetimePattern) throws ParseException {
+        if (datetimePattern == null) {
+            throw new ParseException("The resource datetime pattern is null", -1);
+        }
+        SimpleDateFormat sdf = new SimpleDateFormat("MM/dd/yyyy HH:mm");
```

Comment:
I think with this format string there would be no problems, but adding a locale won't hurt.

The choice of format string seems to stem from the OhlvcItem representation, if this is an externally used format I think this would justify not using ISO format, otherwise I agree that it would be preferable. But in the end a question of preference.

Javafx properties are limited to the javafx thread so I don't know if it applies here.
For formatting (as opposed to parsing) i think introducing CSS styleables would be a good idea independently of this PR.

Reply:
The format was removed. It was used just internally for FinancialTestUtils.